src/SonarScanner.MSBuild.PreProcessor/EngineResolution/EngineResolver.cs

```diff
@@ -26,6 +26,24 @@ namespace SonarScanner.MSBuild.PreProcessor.EngineResolution;
 public class EngineResolver : IResolver
 {
     private const string ScannerEngine = "Scanner Engine";
+    private const string NewBootstrappingEnabledTelemetryKey = "ScannerEngine.NewBootstrapping";
+    private const string ScannerEngineDownloadTelemetryKey = "ScannerEngine.Download";
```

Comment:
Can you create a TelemetryKeys class with these constants and add all keys (like also e.g. [dotnetenterprise.s4net.serverInfo.region](https://github.com/SonarSource/sonar-scanner-msbuild/blob/78d645c2d652e45d3c55ac2b37285af49b88f7fd/src/SonarScanner.MSBuild.Common/Telemetry/TelemetryUtils.cs?plain=1#L43C1-L43C172)) from the scanner there? We should have a single place per source where all the keys are defined.

Reply:
Done in next PR https://github.com/SonarSource/sonar-scanner-msbuild/pull/2775